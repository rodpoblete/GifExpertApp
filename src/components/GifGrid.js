import React from 'react'
import { useFetchGifs } from '../hooks/useFetchGifs';
// import { GifGridItem } from './GifGridItem';

export const GifGrid = ({category}) => {

    const {loading} = useFetchGifs();


    return (
        <>
            <h3>{category}</h3>

            {loading ? 'Cargando...' : 'Data cargada'}

            {/* <div className="card-grid">
                {
                    images.map(img => (
                        <GifGridItem 
                            key={img.id}
                            {... img}
                        />
                    ))
                }
            </div> */}
        </>
    )
}