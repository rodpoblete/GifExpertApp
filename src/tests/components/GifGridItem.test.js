import {GifGridItem} from '../../components/GifGridItem'
import {shallow} from 'enzyme'

describe('Prueba en el componente <GifGridItem />', () => {

    test('Prueba que el componente se muestre correctamente', () => {
        
        const title = 'Un t√≠tulo';
        const url = 'https://localhost/lago.jpg';

        const wrapper = shallow(<GifGridItem title={title} url={url}/>)
        expect(wrapper).toMatchSnapshot();
    })
    
})
