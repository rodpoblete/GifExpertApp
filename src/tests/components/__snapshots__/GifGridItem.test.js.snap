// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en el componente <GifGridItem /> Prueba que el componente se muestre correctamente 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="Un título"
    src="https://localhost/lago.jpg"
  />
  <p>
    Un título
  </p>
</div>
`;
